### DISCUSSION TITLE
Rename `VideoPlayer` react component `Player`?

### DISCUSSION THREAD

**Discussion by adamsoffer** [livepeer/livepeer.js#32] [Ideas]
For simplicity and to better reflect the name of the player ("Livepeer Player") what are our thoughts on renaming the react component `Player` instead of `VideoPlayer`?

**Comment by 0xcadams** [comment]
I like that idea - the only counter-argument would be if we decide to add an audio-only player, and distinguishing the two from each other. I know audio-only use-cases have been discussed, but curious how popular they are.

**Reply by adamsoffer** [reply]
Good point. FYI we do support audio only streams right now. You can simply add `?video=false` to the end of an .m3u8 URL. We could probably abstract this by adding support for an `audioOnly` prop on the component.

But yeah Catalyst does support a few audio-only formats such as mp3, so an audio specific player that renders an audio element as opposed of video element is probably something we'll want to entertain down the road.

If we want to unify a single Player component that supports both video and audio container formats we could potentially detect the format from the source url and then conditionally render a `video` vs `audio` element. This might introduce some unnecessary complexity though compared to simply maintaining two separate `VideoPlayer` and `AudioPlayer` components. I'm sure the html spec went with two distinct elements for a reason. Something to think about anyway 🤔

**Reply by 0xcadams** [reply]
This was done in https://github.com/livepeer/livepeer.js/pull/42!
